examples/hello/bpWriter/helloBPWriter.cpp

```diff
@@ -65,7 +64,9 @@ int main(int argc, char *argv[])
 
         /** Create bp file, engine becomes unreachable after this*/
         bpFileWriter.Close();
-        std::cout << "Wrote file " << filename << " to disk. It can now be read by running ./bin/hello_bpReader.\n";
+        std::cout << "Wrote file " << filename
```

Comment:
`if(rank == 0) { std::cout << ...}` here too keep couts in rank 0. Thanks for adding a nicer message for running this example.

Reply:
@williamfgc : Nice catch; this one is unambiguously wrong. You might have to explain to me in person why the output to `std::cerr` needs to be bound to rank zero, because I still haven't wrapped my mind around it. It doesn't help that MPI and C++ exceptions [aren't friends](https://stackoverflow.com/questions/3762728/am-i-allowed-to-throw-an-exception-inside-mpi-parallelized-code).

Reply:
Yes, MPI doesn't want to be friend with any one :) ,   it's just so when running large jobs you don't want the same message "n" times.